Controllers/cart.js

```diff
@@ -0,0 +1,66 @@
+const Cart = require('../Database/cart');
+const _ = require('lodash');
+
+exports.getCartItems = async (req, res) => {
+	const { cart } = req;
+	try {
+		await cart.populate('cartItems.product').execPopulate();
+		res.json({ success: true, response: cart });
```

Comment:
I feel success: true or false is redundant as we are already sending the status code.

Reply:
Changed!